scripts/gulpfiles/test_tasks.js

```diff
@@ -165,7 +165,11 @@ function compareSize(file, expected) {
   const stat = fs.statSync(name);
   const size = stat.size;
 
-  if (size > compare) {
+  if (!compare) {
+    const message = `Failed: Previous size of ${name} is undefined.`;
+    console.log(`${BOLD_RED}${message}${ANSI_RESET}`);
+    return 1;
+  } else if (size > compare) {
```

Comment:
No need for the else when we're returning!

Reply:
Good call, done and updated the existing clause as well.